packages/binding-coap/README.md

```diff
@@ -26,18 +26,15 @@ The Thing Description is located under the following CoAP URI `coap://plugfest.t
 
 ```js
 // example-client.js
-const { Servient, Helpers } = require("@node-wot/core");
+const { Servient } = require("@node-wot/core");
 const { CoapClientFactory } = require("@node-wot/binding-coap");
 
 // create Servient and add CoAP binding
 const servient = new Servient();
 servient.addClientFactory(new CoapClientFactory());
 
-const wotHelper = new Helpers(servient);
-wotHelper
-    .fetch("coap://plugfest.thingweb.io:5683/testthing")
+WoT.requestThingDescription("coap://plugfest.thingweb.io:5683/testthing")
```

Comment:
I think with the new approach, the `servient` needs to be started first t get the `WoT` object, right?

```ts
servient.start().then(async (WoT) => {
    const td = WoT.requestThingDescription("coap://plugfest.thingweb.io:5683/testthing");
```

Reply:
Good point, will fix soon.. I simply string replaced it 🙈